### PR TITLE
Adding phpIniOptions docs.

### PR DESCRIPTION
**Pull request by sk33lz**
This section was missing form the Environment Configuration page. https://docs.probo.ci/build/environment-configuration/

### CODE REVIEW

File: docs/build-configuration/environment-configuration.md [hunk 1]
```diff
@@ -21,10 +21,20 @@ allowAccessWhileBuilding: true
...
-You can set up basic authentication for your Probo environments with `basicAuth`. This will require users to enter the correct username and password to access the Probo site.
+You can set up Basic Authentication for your Probo builds with `basicAuth`. This will require users to enter the correct username and password to access the Probo build site.
 
 {% highlight yaml %}
 basicAuth:
   username: foo
   password: bar
 {% endhighlight %}
+
+## Modify PHP Environment Options in a Build
```
Comment (jkaeser) [inline docs/build-configuration/environment-configuration.md:32]: @sk33lz Calling out that the `phpIniOptions` can be paired with Drupal and Wordpress in this paragraph might be accidentally misleading. I feel like it implies it _only_ works for these types of sites. I am leaning towards removing this last sentence. What do you think?

**Review by jkaeser — COMMENTED** [review]
@sk33lz I have one question for you.

File: docs/build-configuration/environment-configuration.md (hunk 1 above)
Comment (sk33lz) [reply docs/build-configuration/environment-configuration.md:32]: Updated the language to be a bit more generic to just point to the plugins page.

**Review by jkaeser — APPROVED** [review]
+1

**Pull request merged**